### DISCUSSION TITLE
Typeahead search for link destinations is broken

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#735] [0. General bugs and questions] [answered]
When I add a link, the typeahead search to find the topic and section is no longer working.

Previously, when I selected the option to create a link, I could start typing the topic title to narrow the list so that I could find the one that I wanted.

Now, typing the topic title does nothing. I have to scroll down through the list of topics to find the one that I want, and they are not in any particular order.

**Comment by SamyPesse** [comment]
Hi @JaniceManwiller, thank you for catching this one; it's a regression in our release from 1h ago; a fix is on the way and should be live in 30 minutes.

**Reply by JaniceManwiller** [reply]
Thanks!

**Reply by SamyPesse** [reply, marked as answer]
The fix has been released, you can get it by hard-reloading the app.
